SafeExamBrowser.SystemComponents/SystemInfo.cs

```diff
@@ -155,5 +158,27 @@ private void InitializeMacAddress()
 				}
 			}
 		}
+		private void InitializePnPDevices()
```

Comment:
Please add an empty line between the two methods (as well as between Architecture() and InitializeMacAddress()).